doc/source/changes/version_0_22.rst.inc

```diff
@@ -76,6 +76,19 @@
 
   even if this could suffer, to a lesser extent, the same problem as above when stacking many arrays.
 
+* updated AxisCollection.replace so as to replace one, several or all axes.
+
+* implemented reindex_axis allowing to change order of labels of an axis and/or add new ones (closes :issue:`18`).
```

Comment:
we need a mention of .reindex and in fact, I am not sure having .reindex_axis is worth it. I would rather add the "provide labels instead of an Axis" part of reindex_axis to .reindex, instead of having an extra method. This would have the added benefit that you could write stuff like:
```python
ndtest((2, 2)).reindex(a=['a1', 'a2', 'a0'], b=['b1', 'b0'], fill_value=-1)
```